en/stacks/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: What is Ubuntu Stacks?
+---
+
+# What is Ubuntu Stacks?
+
+Ubuntu Stacks is a set of snap packages that offer system-level functionality that
+enables important core system-level functionality not offered by the base Ubuntu Core
+system. With little to no additional effort, these snaps can be dropped in place
+on one or more new or existing systems bringing significant new functionality.
```

Comment:
I don't mean to be pedantic, but functionality is used three times here with slightly different meaning. Is there a more concrete way to phrase this?